src/mscorlib/src/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -359,14 +354,44 @@ private unsafe SortKey CreateSortKey(String source, CompareOptions options)
                 throw new ArgumentException(SR.Argument_InvalidFlag, nameof(options));
             }
 
-            throw new NotImplementedException();
+            byte [] keyData = null;
+            if (source.Length == 0)
+            { 
+                keyData = EmptyArray<Byte>.Value;
```

Comment:
`EmptyArray<byte>.Value` is not going to compile in CoreRT. Could you please use `Array.Empty<byte>()` instead?